src/pages/index.page.tsx

```diff
@@ -45,34 +45,36 @@ const Home: NextPage = () => {
 
   return (
     <Layout user={user}>
-      <PageLoader isLoading={userLoading || surveyLoading}>
-        <div className="w-full max-w-3xl mx-auto text-white">
-          <section className="mb-36">
-            <div className="text-xs font-extrabold uppercase">
-              {format(currentDate, 'cccc, LLLL d')}
-            </div>
-            <div className="text-4xl font-extrabold">Today</div>
-          </section>
-          {store.surveys.length !== 0 ? (
-            <section className="flex items-center">
-              {store.surveys.map((survey) => (
-                <div className="mx-4" key={survey.id}>
-                  <img
-                    src={survey.coverImageUrl}
-                    width={280}
-                    height={70}
-                    alt={`${survey.id} survey image`}
-                  />
-                  <h2>{survey.title}</h2>
-                  <div>{survey.description}</div>
-                </div>
-              ))}
+      <div className="w-full max-w-3xl mx-auto text-white">
+        <SurveyListSkeleton isLoading={surveyLoading}>
+          <>
+            <section className="mb-36">
+              <div className="text-xs font-extrabold uppercase">
+                {format(currentDate, 'cccc, LLLL d')}
+              </div>
+              <div className="text-4xl font-extrabold">Today</div>
             </section>
-          ) : (
-            <SurveyListPlaceholder />
-          )}
-        </div>
-      </PageLoader>
+            {store.surveys.length !== 0 ? (
+              <section className="flex items-center">
+                {store.surveys.map((survey) => (
+                  <div className="mx-4" key={survey.id}>
+                    <img
+                      src={survey.coverImageUrl}
+                      width={280}
+                      height={70}
+                      alt={`${survey.id} survey image`}
+                    />
+                    <h2>{survey.title}</h2>
+                    <div>{survey.description}</div>
+                  </div>
+                ))}
+              </section>
+            ) : (
+              <SurveyListPlaceholder />
+            )}
```

Comment:
Should we bring this block into a constant instead? Then we can read the code easier

Reply:
As discussed this will be fixed in next PR